lib/Saml2/Utils.php

```diff
@@ -464,9 +525,19 @@ public static function isHTTPS()
      */
     public static function getSelfURLNoQuery()
     {
-
         $selfURLhost = self::getSelfURLhost();
-        $selfURLNoQuery = $selfURLhost . $_SERVER['SCRIPT_NAME'];
+
+        if (!empty(self::getBaseURLPath())) {
```

Comment:
you can't use `empty` with method in php < 5.5 so you need to assign a var first as the module supports 5.3+